Sources/Swift/Integrations/SessionReplay/SentryReplayEvent.swift

```diff
@@ -0,0 +1,48 @@
+@_implementationOnly import _SentryPrivate
+import Foundation
+
+@objcMembers
+class SentryReplayEvent: Event {
```

Comment:
Compared to the obj-c impl trace_ids and error_ids are missing, were they not used?